vgen/src/main/java/io/gapi/vgen/ruby/RubyGapicContext.java

```diff
@@ -184,6 +207,53 @@ private String returnTypeComment(Method method) {
   }
 
   /**
+   * Return comment lines for a given proto element.
+   */
+  public List<String> getDocComment(ProtoElement element) {
+    StringBuilder contentBuilder = new StringBuilder();
+    if (element.hasAttribute(ElementDocumentationAttribute.KEY)) {
+      String text = RDocCommentFixer.rdocify(DocumentationUtil.getScopedDescription(element));
+      text = text.trim();
+      contentBuilder.append(text.replaceAll("[ \\t\\r\u000B]*\\n", "\n"));
```

Comment:
Considering about this, moved this logic -- trimming and replacing -- to RDocCommentFixer, because it's always good to be done.